pixi_diff_to_markdown/models.py

```diff
@@ -67,6 +67,9 @@ def transform_before(cls, data: Any) -> Any:
             )
 
         if "pypi" in data:
+            if data["pypi"].startswith("git+https://"):
+                # in case of a direct git dependency, we don't have a version
+                data["version"] = "#.#.#"
```

Comment:
I would just explicitly set it to `None`, this seems a bit more sensible than a random `#.#.#` string

Instead, we could just adjust the `validate_after` method to not raise an assertion error anymore

Reply:
I actaully tried that initaily, but additional things were breaking down the line,
and it started to become too complex

when running `pixi list`, the version for such dependencies retured as `0.0.0`
```
microsoft_python_type_stubs  0.0.0                                               pypi   git+https://github.com/microsoft/python-type-stubs.git#dcb5c8c2a61a2e4653b4becc9f20f6c42858a701
```

perhaps we should set it here to `0.0.0` as well to be consistent with `pixi list`?

Reply:
I've updated the code (and tests) to support `0.0.0`
let me know if you prefer to set it to `None`, and I'll try to do that instead

Reply:
i pushed to your branch and used `None`: [`1080afb` (#57)](https://github.com/pavelzw/pixi-diff-to-markdown/pull/57/commits/1080afbc3bc1e3db47551157c54c4ddec4e50834)